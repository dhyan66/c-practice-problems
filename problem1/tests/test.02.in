new 30
set 1
set 3
set 5
toggle 3
set 10
toggle 11
set 11
toggle 3
