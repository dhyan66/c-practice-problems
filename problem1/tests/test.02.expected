00 00 00 00 
02 00 00 00 
0a 00 00 00 
2a 00 00 00 
22 00 00 00 
22 04 00 00 
22 0c 00 00 
22 0c 00 00 
2a 0c 00 00 
